### PR TITLE
Refresh button to choose nearby places from marker location

### PR DESCRIPTION
**Pull request by kkostrubiec**
Hello, this commit contains:

- add refresh button to choose nearby places from marker location,
- move marker to selected location from search list,
- fixed build sample app, needed to add multidex.

Please review and let me know what do You think.

### CODE REVIEW

File: library/src/main/java/com/rtchagas/pingplacepicker/ui/PlacePickerActivity.kt [hunk 1]
```diff
@@ -114,6 +121,12 @@ class PlacePickerActivity : AppCompatActivity(), PingKoinComponent,
         // Initializes the map
         val mapFragment = supportFragmentManager.findFragmentById(R.id.map) as SupportMapFragment
         mapFragment.getMapAsync(this)
+
+        compositeDisposable.addAll(
```
Comment (rtchagas) [inline library/src/main/java/com/rtchagas/pingplacepicker/ui/PlacePickerActivity.kt:125]: You should clear the compositeDisposable in the onDestroy()

Do you know RxView.Clicks?
https://stackoverflow.com/questions/32498210/debouncing-button-clicks-using-rx

It does exactly this but in a more straightforward way. :)

I should've done that before but I forgot. Thanks for pointing this.

**Review by rtchagas — CHANGES_REQUESTED** [review]

**Review by rtchagas — APPROVED** [review]
Good improvement!

**Review by rtchagas — COMMENTED** [review]
I commented more about this in the #26

**Review by rtchagas — CHANGES_REQUESTED** [review]
- Please remove the first commit as it will be addressed by /nearbysearch in Maps API.
- The second commit is perfect.
- In the third one, could you please try RxView.clicks?

**Comment by rtchagas** [comment]
Hi @kkostrubiec 

Please rebase your branch with my latest changes.
I really want to pull the click debounce and the click area improvements to the next release.

Thank you!

**Comment by kkostrubiec** [comment]
Hi @rtchagas, I will fix it as fast as I can :)

**Pull request closed without merging**